includes/Admin/Customizer.php

```diff
@@ -124,6 +125,21 @@ public function register() {
 		add_action( 'customize_register', [ $this, 'register_customizer_settings' ] );
 	}
 
+	/**
+	 * Check whether the conditional object is currently needed.
+	 *
+	 * @since 1.16.0
+	 *
+	 * @return bool Whether the conditional object is needed.
+	 */
+	public static function is_needed(): bool {
+		if ( function_exists( '\wp_is_block_theme' ) ) {
+			return ! wp_is_block_theme();
+		}
+
+		return true;
```

Comment:
What was wrong with `return ! function_exists( 'wp_is_block_theme' ) || ! wp_is_block_theme();`?

Reply:
Readability.

Reply:
I found it quite readable 🤷